C++/Navio/Common/gpio.cpp

```diff
@@ -9,12 +9,17 @@
 #include <cstdio>
 #include <cstdlib>
 #include <cstring>
+#include <fstream>
+#include <string>
+#include <iostream>
 
 #include "gpio.h"
 
 #define LOW                 0
 #define HIGH                1
 
+#define ERROR               1
```

Comment:
Way too common naming, remember, it's a preprocessor, we can already have `ERROR` define somewhere. Make more concrete naming to avoid confuses.

Reply:
Would `EXIT_ERROR` be better?

Reply:
Yes, better, but if you bind it with our product name and/or process name (we're getting RPi revision) it will be definitely safe and won't be used by mistake.

Reply:
Good! I changed it to `RPI_MODEL_ERROR`, I think it seems logical.